orion/sites/admin/src/index.js

```diff
@@ -15,13 +15,16 @@ import { ProgramEditPage } from './programs/programEdit.js';
 import { AchievePage } from './achieve/achieve.js';
 import { AchieveEditPage } from './achieve/achieveEdit.js';
 import { AnnouncePage } from './announce/announce.js';
+import { AnnounceEditPage } from './announce/announceEdit.js';
 import { LocationPage } from './location/location.js';
 import { LocationEditPage } from './location/locationEdit.js';
 
 const Achieve = () => <AchievePage/>;
 const AchieveEdit = () => <AchieveEditPage/>;
 const AchieveEditMatch = ({match}) => <AchieveEditPage year={match.params.year}/>;
 const Announce = () => <AnnouncePage/>;
+const AnnounceEdit = () => <AnnounceEditPage/>;
+const AnnounceEditMatch = ({match}) => <AnnounceEditPage programId={match.params.AnnounceId}/>;
```

Comment:
looks like you put `programId=...` should probably be something else

Reply:
the `params` part should match with the variable name below on line 57.
So if the path is `"/announce/:announceId/edit"`, the variable is `announceId`. Lower-cased not Upper-cased.